src/app/components/header/header.component.html

```diff
@@ -1,7 +1,20 @@
-<div class="container w-50 my-5">
-  <div class="header text-center">
-    <h1>
-      <u><b>ADIVINA LA PALABRA</b></u>
-    </h1>
+<div class="header text-center">
+    <nav class="navbar navbar-expand-lg navbar-light navigation ">
+      <div class="collapse navbar-collapse d-flex justify-content-center" id="navbarNav">
+        <ul class="navbar-nav custom justify-content-center custom">
+          <li *ngIf="checkRoute() == 'inicio'" class="nav-item active">
```

Comment:
```suggestion
          <li *ngIf="selectedRoute === HOME" class="nav-item active">
```
use var to select the actual view and constants to every route. In addition to this, never use double equal '==' instead of triple equal symbol '==='